perun-base/src/main/java/cz/metacentrum/perun/core/api/GroupResourceStatus.java

```diff
@@ -6,8 +6,19 @@
  * @author Radoslav Čerhák <r.cerhak@gmail.com>
  */
 public enum GroupResourceStatus {
-	ACTIVE,
-	INACTIVE,
-	FAILED,
-	PROCESSING
+
+	ACTIVE(3),
+	INACTIVE(2),
+	FAILED(1),
+	PROCESSING(0);
```

Comment:
Maybe I would give PROCESSING level 2, because it is "closer" to being ACTIVE. It might be more interesting for users to know that one of the group-resource assignments is being activated. What do you think, @metodej @Johaney-s?

Reply:
@cuadradek I think the PROCESSING state should be the least important one because you can't rely on it for it is only temporary, plus it could end up as FAILED anyway. I believe the final statuses are more important.

Reply:
Now that I think about it, even though it could end up FAILED, it could end up ACTIVE, making the present INACTIVE/FAILED state temporal. It could be beneficial for users to know they should wait a moment and check later. It makes sense for me to increase its _importance_.

Reply:
Alright, then please change the values accordingly.